### PR TITLE
fix: kube-lint issues

### PR DESCRIPTION
**Pull request by yftacherzog**

### CODE REVIEW

File: dependencies/tekton-results/kustomization.yml [hunk 1]
```diff
@@ -26,6 +26,27 @@ patches:
     target:
       kind: StatefulSet
       name: tekton-results-postgres
+  - patch: |
+      - op: add
+        path: /metadata/annotations/ignore-check.kube-linter.io~1liveness-port
```
Comment (avi-biton) [inline dependencies/tekton-results/kustomization.yml:31]: Shouldn't it be `/metadata/annotations/ignore-check.kube-linter.io/liveness-port` ?

Comment (yftacherzog) [reply dependencies/tekton-results/kustomization.yml:31]: `~1` is the escape character for `/` (I'm not sure this is for YAML, or for something else but this is what works in this case), so this eventually translates to:
```yaml
metadata:
  annotations:
    ignore-check.kube-linter.io/liveness-port: <what's in value>
```

Comment (avi-biton) [reply dependencies/tekton-results/kustomization.yml:31]: I checked it myself and it seems to work fine.
Thanks for the explanation.

**Review by avi-biton — APPROVED** [review]
lgtm

**Pull request merged**